code/game/gamemodes/clock_cult/clock_ratvar.dm

```diff
@@ -122,10 +122,10 @@
 	clockwork_desc = "<span class='large_brass'><b><i>Ratvar, the Clockwork Justiciar, your master eternal.</i></b></span>"
 	icon = 'icons/effects/512x512.dmi'
 	icon_state = "ratvar"
-	pixel_x = -248
-	pixel_y = -248
+	pixel_x = -235
+	pixel_y = -235
```

Comment:
The dimensions on the sprite were exactly the same, so I figured that changing both would make it better overall.